pyCGM_Single/clusterCalc.py

```diff
@@ -1,13 +1,41 @@
+"""This file is used for calculating cluster transformations to apply
+in the case of a missing marker.
+
+It is used in:
+pycgmStatic.py - lines 105-115
```

Comment:
This should be a bit more descriptive.  Originally it was as a note, but with other files being changed with docstrings these values are likely to change, so at least include function names